pkg/deployments/tasks/20220704-timelock-authorizer/input/mainnet.ts

```diff
@@ -0,0 +1,250 @@
+const EMERGENCY_SUBDAO_MULTISIG = '0xa29f61256e948f3fb707b4b3b138c5ccb9ef9888';
+const BLABS_OPS_MULTISIG = '0x02f35dA6A02017154367Bc4d47bb6c7D06C7533B';
+const BLABS_VEBAL_MULTISIG = '0xd2eb7bd802a7ca68d9acd209bec4e664a9abdd7b';
+const GAUNTLET_FEE_SETTER = '0xe4a8ed6c1d8d048bd29a00946bfcf2db10e7923b';
+
+export const root = DAO_MULTISIG;
+
+const batchRelayerPermissions = createRoleData(BalancerRelayer.output().BalancerRelayer, Vault.output().Vault, [
+  Vault.actionId('Vault', 'setRelayerApproval(address,address,bool)'),
+  Vault.actionId(
+    'Vault',
+    'batchSwap(uint8,(bytes32,uint256,uint256,uint256,bytes)[],address[],(address,bool,address,bool),int256[],uint256)'
+  ),
+  Vault.actionId('Vault', 'joinPool(bytes32,address,address,(address[],uint256[],bytes,bool))'),
+  Vault.actionId(
+    'Vault',
+    'swap((bytes32,uint8,address,address,uint256,bytes),(address,bool,address,bool),uint256,uint256)'
+  ),
+  Vault.actionId('Vault', 'exitPool(bytes32,address,address,(address[],uint256[],bytes,bool))'),
+  Vault.actionId('Vault', 'manageUserBalance((uint8,address,uint256,address,address)[])'),
+]);
+
+const lidoRelayerPermissions = createRoleData(LidoRelayer.output().LidoRelayer, Vault.output().Vault, [
+  Vault.actionId(
+    'Vault',
+    'batchSwap(uint8,(bytes32,uint256,uint256,uint256,bytes)[],address[],(address,bool,address,bool),int256[],uint256)'
+  ),
+  Vault.actionId('Vault', 'joinPool(bytes32,address,address,(address[],uint256[],bytes,bool))'),
+  Vault.actionId(
+    'Vault',
+    'swap((bytes32,uint8,address,address,uint256,bytes),(address,bool,address,bool),uint256,uint256)'
+  ),
+  Vault.actionId('Vault', 'exitPool(bytes32,address,address,(address[],uint256[],bytes,bool))'),
+  Vault.actionId('Vault', 'manageUserBalance((uint8,address,uint256,address,address)[])'),
+]);
+
+const gnosisProtocolRelayerPermissions = createRoleData(GnosisProtocolRelayer, Vault.output().Vault, [
+  Vault.actionId(
+    'Vault',
+    'batchSwap(uint8,(bytes32,uint256,uint256,uint256,bytes)[],address[],(address,bool,address,bool),int256[],uint256)'
+  ),
+  Vault.actionId('Vault', 'manageUserBalance((uint8,address,uint256,address,address)[])'),
+]);
+
+const protocolFeesPermissions: RoleData[] = flatten([
+  createRoleData(
+    ProtocolFeesWithdrawer.output().ProtocolFeesWithdrawer,
+    ProtocolFeesCollector.output().ProtocolFeesCollector,
+    [Vault.actionId('ProtocolFeesCollector', 'withdrawCollectedFees(address[],uint256[],address)')]
+  ),
+  createRoleData(TREASURY_MULTISIG, ProtocolFeesWithdrawer.output().ProtocolFeesWithdrawer, [
+    ProtocolFeesWithdrawer.actionId('ProtocolFeesWithdrawer', 'withdrawCollectedFees(address[],uint256[],address)'),
+  ]),
+  createRoleData(EMERGENCY_SUBDAO_MULTISIG, ProtocolFeesWithdrawer.output().ProtocolFeesWithdrawer, [
+    ProtocolFeesWithdrawer.actionId('ProtocolFeesWithdrawer', 'denylistToken(address)'),
+  ]),
+]);
+
+const veBALPermissions: RoleData[] = flatten([
```

Comment:
We're missing the bridge management permissions (IArbitrumFeeProvider.setArbitrumFees and IOptimismGasLimitProvider.setOptimismGasLimit)